### DISCUSSION TITLE
How do I get to the intensity value of Unique proteins (not protein groups)?

### DISCUSSION THREAD

**Discussion by bluemooninvestor** [vdemichev/DiaNN#1115] [Q&A]
I want to analyze the output to find this:

(1) What are the unique proteins that have been identified as a list (not protein groups, I want to find the list of proteins identified using proteotypic peptides)
(2) The unique and shared peptides details of aforementioned unique proteins
(3) The intensity of these unique and shared peptides, so that I can do protein intensity summarization based on my method of choice
(4) Is there any way to dictate the MaxLFQ calculation to use only proteotypic peptides in DIA-NN

For 1-3, I know I have to use some scripting. I am only asking where should I look for these data.

Kindly guide

**Comment by vbastos18** [comment]
Hello, 

You can answer all these questions with this Shiny script that works with the report output file of DIA-NN as a starting point : 

https://github.com/marseille-proteomique/DIAgui

Hope it helps :)

**Reply by vdemichev** [reply]
In case you want to use this directly via R commands, here's how:
(1) 
- Unique genes: unique(df$Genes[df$Proteotypic == 1 & df$Protein.Q.Value <= 0.01])
- Unique protein isoforms: analyse with protein inference set to isoforms, then unique(df$Protein.Group[df$Proteotypic == 1 & df$Protein.Q.Value <= 0.01])

(2) Peptides with Proteotypic == 1 are unique
(3) Precursor.Normalised of the respective precursor
(4) No, need to reproduce in R (iq R package) or, if don't want to do any coding, can use DIAgui recommended above.

**Comment by bluemooninvestor** [comment]
Thank you for the detailed help!
